package.json

```diff
@@ -0,0 +1,84 @@
+{
+    "name": "@oat-sa/tao-item-runner-qti",
+    "version": "0.1.0",
+    "displayName": "TAO Item Runner QTI",
+    "description": "QTI Item Runner frontend library of TAO",
+    "files": [
+        "dist",
+        "src"
+    ],
+    "directories": {
+        "test": "test"
+    },
+    "scripts": {
+        "test:keepAlive": "node ./build/webserver.js",
+        "test": "node ./build/testrunner.js",
+        "build": "rollup --config ./build/rollup.config.js",
+        "buildScss": "node ./build/scss.js",
+        "lint": "eslint src test",
+        "prepare": "npm run buildScss && npm run build"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/oat-sa/tao-item-runner-qti-fe.git"
+    },
+    "keywords": [
+        "tao",
+        "item",
+        "runner",
+        "qti",
+        "fe",
+        "frontend"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "license": "GPL-2.0",
+    "bugs": {
+        "url": "https://github.com/oat-sa/tao-item-runner-qti-fe/issues"
+    },
+    "homepage": "https://github.com/oat-sa/tao-item-runner-qti-fe#readme",
+    "devDependencies": {
+        "@csstools/postcss-sass": "^4.0.0",
+        "@oat-sa/browserslist-config-tao": "^0.1.0",
+        "@oat-sa/tao-core-libs": "^0.1.0",
+        "@oat-sa/tao-core-sdk": "^0.3.0",
+        "@oat-sa/tao-core-ui": "github:oat-sa/tao-core-ui-fe#feature/NEX-131_update-ui-modules-from-tao-core",
```

Comment:
This branch does not exist anymore.

Reply:
Fixed.
And I still need to update the package.json reference to `tao-item-runner` when that one gets merged and released on npm.